packages/dcos-diagnostics/extra/dcos-diagnostics-master.service

```diff
@@ -12,6 +12,8 @@ PermissionsStartOnly=True
 User=dcos_diagnostics
 # Allow r/w access to the socket file
 SupplementaryGroups=dcos_adminrouter
+ExecStartPre=/usr/bin/mkdir -p ${DCOS_DIAGNOSTICS_BUNDLE_DIR}
+ExecStartPre=/usr/bin/chown dcos_diagnostics ${DCOS_DIAGNOSTICS_BUNDLE_DIR}
```

Comment:
I think this is what's causing `teamcity/dcos/test/azure/arm` to fail. We can't assume `mkdir` and `chown` are at these locations, since they're elsewhere on Ubuntu. You'll need to wrap these commands in a `bash -c` so they get looked up in `$PATH`. See #3405.

Reply:
`/bin/bash` might not be available on other systems shouldn't we use `/bin/sh`?

Reply:
That's worth considering. It's not a problem right now since every OS we support provides `/bin/bash`, and we already assume it's available on DC/OS nodes in several other places in the codebase. I do like being certain that we're using bash, and not some other shell that the OS decided to place at `/bin/sh` (it's dash on Ubuntu).